packages/frontend/src/Build.svelte

```diff
@@ -632,8 +638,32 @@ export function goToHomePage(): void {
                 ><Fa icon="{showAdvanced ? faCaretDown : faCaretRight}" class="inline-block mr-1" /> Advanced Options
               </span>
               {#if showAdvanced}
+                <!-- Build config -->
+                <div class="mb-2">
+                  <label for="buildconfig" class="block mb-2 text-md font-semibold">Build config</label>
+                  <div class="flex flex-row space-x-3">
+                    <Input
+                      name="buildconfig"
+                      id="buildconfig"
+                      bind:value="{buildConfigFile}"
+                      placeholder="Build configuration file (config.toml or config.json)"
+                      class="w-full"
+                      aria-label="buildconfig-select" />
+                    <Button on:click="{() => getBuildConfigFile()}">Browse...</Button>
+                  </div>
+                  <p class="text-xs text-[var(--pd-content-text)] pt-2">
+                    The build configuration file is a TOML or JSON file that contains the build options for the disk
+                    image. Customizations include user, password, SSH keys and kickstart files. More information can be
+                    found in the <Link
+                      externalRef="https://github.com/osbuild/bootc-image-builder?tab=readme-ov-file#-build-config"
+                      >bootc-image-builder documentation</Link
+                    >.
+                  </p>
+                </div>
+
+                <!-- AWS -->
                 <div>
-                  <span class="text-sm font-semibold mb-2 block">Upload image to AWS</span>
+                  <span class="text-md font-semibold mb-2 block">Upload image to AWS</span>
```

Comment:
I don't think text-md is defined? May be just defaulting to text-base or whatever the parent is here.